### PR TITLE
Support configurable classpath

### LINKED ISSUE #25: Make classpath configurable
Add configuration option so that user can add to classpath.

Relates to #1

### PR DESCRIPTION
**Pull request by daw42**
Support classpath via user configuration and/or environment variable.   Uses OS dependent classpath delimiter.

 - Fixes #25 
 - Includes content from #26

### CODE REVIEW

**Comment by steelbrain** [comment]
@daw42 how about we merge that PR first and you base this PR on it?

File: lib/init.coffee [hunk 1]
```diff
@@ -8,12 +8,19 @@ module.exports =
       type: 'string'
       title: 'Path to the javac executable'
       default: 'javac'
+    classpath:
+      type: 'string'
```
Comment (k2b6s9j) [inline lib/init.coffee]: Could this possibly be an array? See linter-clang: https://github.com/AtomLinter/linter-clang/blob/master/lib/main.js#L114-L116

**Comment by k2b6s9j** [comment]
I see no problem merging this and closing #26.

**Comment by steelbrain** [comment]
@k2b6s9j We should appreciate community contributions instead of closing them :)
@daw42 Be advised that this PR has some merge conflicts

**Comment by daw42** [comment]
I'll close this and open a new one.

**Comment by steelbrain** [comment]
You didn't have to close it, try `merge-conflicts` atom package.

**Comment by daw42** [comment]
Ok, I'll fix the conflicts.

**Comment by daw42** [comment]
Fixed, thanks!

**Comment by steelbrain** [comment]
I am not in a position to test this any time soon, @k2b6s9j You're our only hope :D

File: lib/init.coffee (hunk 1 above)
Comment (daw42) [inline lib/init.coffee]: I think we should stick with a string.  Java programmers are used to specifying the classpath as a single string.

Comment (k2b6s9j) [inline lib/init.coffee]: Fair enough. I haven't done purely Java in quite a while now.

**Pull request merged**